### DISCUSSION TITLE
How to get rowcount  from aggrid and also how save the aggrid to Mysql DB

### DISCUSSION THREAD

**Discussion by bajirama** [zauberzeug/nicegui#1669] [Q&A]
### Question

I am trying to get the row count from the aggrid and also want to save the gird if the row is greater than Zero 
tried the following and it is not working  please help 

```py
grid = ui.aggrid({
    'defaultColDef': {'flex': 1},
    'columnDefs': [
        {'headerName': 'Name', 'field': 'name'},
        {'headerName': 'Age', 'field': 'age'},
    ],
    'rowData': [
        {'name': 'Alice', 'age': 18},
        {'name': 'Bob', 'age': 21},
    ],
})

def get_row_count():
    api = grid.call_api_method('getDisplayedRowCount')
    return api

row_count = get_row_count()
print(row_count)
```

**Comment by natankeddem** [comment]
The `call_api_method` doesn't return anything:
https://github.com/zauberzeug/nicegui/blob/9f7aef66eadfe5ecd602f2c4425acd0d5fd698dc/nicegui/elements/aggrid.py#L69

You can find inspiration for how to access the grid api to return data here:
https://github.com/zauberzeug/nicegui/blob/9f7aef66eadfe5ecd602f2c4425acd0d5fd698dc/nicegui/elements/aggrid.py#L98
